readme.txt

```diff
@@ -0,0 +1,9 @@
+=== Dintero Checkout for WooCommerce ===
+Contributors: krokedil
+Tags: woocommerce, dintero, ecommerce, e-commerce, checkout
+Requires at least: 5.8.3
+Requires PHP: 8.0.10
```

Comment:
@montazarr: Will it not support 7.4?

Reply:
@andersem  It should support PHP 7 since I am not using any PHP 8 features (as far as I can tell), but my development environment is running PHP 8.0.10.